<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Jack Anderson</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="opaqueResize">
          <bool>true</bool>
         </property>
         <property name="handleWidth">
          <number>4</number>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTextEdit" name="txt_dataset">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_dataset">
             <property name="text">
              <string>Dataset...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Classifier One</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_alg1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_alg1">
             <property name="text">
              <string>Select Classifier....</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chk_two_stage">
             <property name="text">
              <string>Two Stage</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Classifier Two</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_alg2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_alg2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Select Classifier...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_run">
             <property name="text">
              <string>Run</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QTabWidget" name="tabWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="tab">
               <attribute name="title">
                <string>Results</string>
               </attribute>
              </widget>
              <widget class="QWidget" name="tab_2">
               <attribute name="title">
                <string>Graphs</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QPushButton" name="findme">
                  <property name="text">
                   <string>PushButton</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
             <widget class="QGroupBox" name="groupBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="title">
               <string>Output</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QPlainTextEdit" name="plainTextEdit">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="text">
                  <string>Clear</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionAlgorithm"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAlgorithm">
   <property name="text">
    <string>Algorithm</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string comment="Ctrl+Alt+s"/>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string comment="Ctrl+Q"/>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_alg1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btn_alg1_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_two_stage</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>chk_two_stage_toggled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_alg2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btn_alg2_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_run</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btn_run_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>593</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_dataset</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btn_dataset_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_dataset_clicked()</slot>
  <slot>btn_alg1_clicked()</slot>
  <slot>chk_two_stage_toggled()</slot>
  <slot>btn_alg2_clicked()</slot>
  <slot>btn_run_clicked()</slot>
 </slots>
</ui>
